bin/navuser

```diff
@@ -76,6 +76,27 @@ def adduser(args):
     print(msg % args.login, file=sys.stderr)
 
 
+def adminify(args):
+    action = args.action
+    try:
+        account = Account.objects.get(login=args.login)
+    except Account.DoesNotExist:
+        print("User %s does not exist" % args.login, file=sys.stderr)
+        sys.exit(1)
+
+    if action == 'make':
+        admin = AccountGroup.objects.get(id=AccountGroup.ADMIN_GROUP)
+        admin.accounts.add(account)
+        msg = "User %s was made an admin" % args.login
+    elif action == 'unmake':
```

Comment:
This is broken. The argument parser supports the arguments `add` and `remove`: This results in error messages `Unknown argument add` or `Unknown argument remove` when the `-a` or `-r` options are used.

Reply:
It's also broken in the sense that: Both the `make` and `unmake` options will *add* the user to the admin group...